int16 EPS_command
uint16 SCC_command
